addons/website_account/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Website Account',
+    'category': 'Website/Website',
+    'summary': 'Bridge module between account and website.',
+    'description': 'Bridge module between account and website.',
+    'depends': ['account', 'website'],
+    'auto_install': True,
+    'data': [
+        'views/snippets.xml',
```

Comment:
@bvr-odoo Even if structurally correct to create a new bridge app for this, it feels overkill... we'll think about another solution (probably more like the previous one which was ugly too but lead to less code to maintain).

Reply:
@bvr-odoo After further discussion, you can actually leave the `customize_show` field on those views (so just the ones for portal) and adapt the previous JS which auto populated the customize menu to now auto populate the right panel.
That will be way easier to maintain (and use for the community).

We will end up with 2 mechanisms:
- Either the view is customize_show and auto populated in the right panel (basic / automatic way)
- Either the view is not customize_show and there is some declared options in XML (advanced way, mainly useful to build a nicer option layout like grouping some options, making it something else than a toggle etc etc).

cc @qsm-odoo